tests/data/test_wrappers.py

```diff
@@ -1,14 +1,18 @@
 """Tests for `imitation.data.wrappers`."""
 
-from typing import List, Sequence
+from typing import List, Sequence, Type, Dict, Optional
 
 import gymnasium as gym
 import numpy as np
 import pytest
 from stable_baselines3.common.vec_env import DummyVecEnv
 
 from imitation.data import types
-from imitation.data.wrappers import BufferingWrapper
+from imitation.data.wrappers import (
+    BufferingWrapper,
+    HumanReadableWrapper,
```

Comment:
Should be split into separate PR.

Reply:
Yes. Sorry for the confusion. I didn't do well on branching. Just fixed it.